src/looprestoration.rs

```diff
@@ -23,3 +29,288 @@ pub union LooprestorationParams {
     pub filter: Align16<[[int16_t; 8]; 2]>,
     pub sgr: LooprestorationParams_sgr,
 }
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+pub struct Dav1dLoopRestorationDSPContext {
+    pub wiener: FnVersion,
+    pub sgr: FnVersion,
```

Comment:
I think we can mark these temporarily `pub` until `looprestoration` template deduplication.  I'll do the same in my PR.